access-checkout/src/main/java/com/worldpay/access/checkout/client/validation/AccessCheckoutValidationInitialiser.kt

```diff
@@ -13,8 +13,17 @@ import com.worldpay.access.checkout.validation.result.handler.ResultHandlerFacto
 import com.worldpay.access.checkout.validation.result.state.CardValidationStateManager
 import com.worldpay.access.checkout.validation.result.state.CvcValidationStateManager
 
+/**
+ * Class that is responsible for initialising a given [ValidationConfig]
```

Comment:
`Class that is responsible for initialising **validation using** a given [ValidationConfig]` ?

Reply:
Done